.github/scripts/automerge.py

```diff
@@ -49,7 +50,7 @@ def is_approved(pr):
     print("No PR was automerged.")
 
 # Now, update all PRs that are behind, regardless of automerge label.
-pulls = fetch_pulls('behind', '')
+pulls = fetch_pulls('behind', {'automerge', 'autoupdate'})
```

Comment:
This looks good, but I wonder, these labels are little ambiguous. Technically, automerge could refer to either action. Maybe "sync-with-master" or something like that?

Reply:
Well, the perspective here is from the PR UI. The button to sync from master is called "Update branch", while the one to merge into master is called "Merge".